Examples/RPC/go/README.md

```diff
@@ -0,0 +1,15 @@
+# Go HTTP RPC Sample
```

Comment:
Please use lowercase and singular for all directory names. This is what we do for proto package names.

Reply:
Lower-casing "Examples", "RPC", and "REST" but keeping "examples" as plural.